scss/components/action-bar.scss

```diff
@@ -122,7 +122,7 @@ $block: #{$fd-namespace}-action-bar;
           }
         }
         .fd-popover {
-            top: - fd-space(1);
+            top: 0;
```

Comment:
Is this just an issue when popovers wrap components (buttons, in this case) in an action bar?  I suppose so simply because of the selector at work.  Is there a reason to keep the declaration with `top: 0` or should this block just be removed entirely?

Reply:
yeah, i tried removed  .fd-popover {top: 0}, and it still works. thanks